.github/workflows/labels.yml

```diff
@@ -0,0 +1,20 @@
+name: Tools
+
+on:
+  workflow_call:
+
+permissions:
+  issues: write
+
+jobs:
+  sync-labels:
+    name: Sync labels
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+
+      - name: Sync labels
+        uses: micnncim/action-label-syncer@3abd5ab72fda571e69fffd97bd4e0033dd5f495c
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
In #39 you specified the secrets in `workflow_call`. Is it correct that this isn't needed for `GITHUB_TOKEN`?

Reply:
That is correct. As per [the docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows):

> When a reusable workflow is triggered by a caller workflow, the github context is always associated with the caller workflow. The called workflow is automatically granted access to github.token and secrets.GITHUB_TOKEN.